### PR TITLE
:turtle: Allow sig-chairs with write access to multiple repos

### PR DESCRIPTION
**Pull request by harshad16**
Allow sig-chairs with write access to multiple repos
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies
Fixes: https://github.com/thoth-station/core/issues/427

### CODE REVIEW

**Comment by goern** [comment]
I wonder if we need this, please see https://github.com/thoth-station/core/issues/427#issuecomment-1209071009

File: community/github-config.yaml [hunk 1]
```diff
@@ -1138,6 +1138,110 @@ orgs:
           user-api: triage
           website-tooling: triage
           workflow-helpers: triage
+      SigChairs:
```
Comment (goern) [inline community/github-config.yaml:1141]: isnt 'write' permission less than the 'triage' that all the devs team members have?

Comment (codificat) [reply community/github-config.yaml:1141]: no, `write` allows more than `triage`: https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role

Comment (goern) [reply community/github-config.yaml:1141]: you are right, I was misinterpreting the table layout

**Comment by harshad16** [comment]
The write access is needed, so sig chairs can edit the issue and its title for grooming and planning.
with access to the project will still not suffice the requirement, we would like chair to easily manage the issues.

**Pull request merged**